presto-native-execution/presto_cpp/main/http/HttpServer.h

```diff
@@ -268,10 +268,15 @@ class HttpsConfig {
 
 class HttpServer {
  public:
+  /// The server will create an IO pool of 'httpExecThreads' number of threads
```

Comment:
Shall we enforce http server always take a http server executor so we can remove httpExecThreads? Thanks!